modules/installation/pages/uyuni-proxy-setup.adoc

```diff
@@ -209,32 +209,25 @@ filename: "pxelinux.0"
 ----
 
 
-////
-// REMARK: 2019-08-23, ke: this needs closer checking
-// see https://github.com/uyuni-project/uyuni-docs/issues/110
-[[replacing.a.susemgrproxy]]
-== Replace a {susemgrproxy}
+
+[[replacing.a.uyuniproxy]]
+== Replace a Uyuni Proxy
 
 A proxy does not contain any information about the clients that are connected to it.
-Because of this, a proxy can be replaced by a new one at any time.
+Therefore, a proxy can be replaced by a new one at any time.
 The replacement proxy must have the same name and IP address as its predecessor.
 
 Leave the old proxy in place while you prepare the replacement.
 Create a reactivation key for this system and then register the new proxy using the reactivation key.
 If you do not use the reactivation key, you will need to re-register all the clients against the new proxy.
 
 [[proxy.migration3.replace]]
-.Procedure: Replacing a proxy and Keeping the Clients Registered
+.Procedure: Replacing a Proxy and Keeping the Clients Registered
 . Before starting the actual migration procedure, save the data from the old proxy, if needed.
-Consider copying important data to a central place that can also be accessed by the new proxy.
-// REMARK: 2019-08-23, ke: this is unclear
-// ** Copy the scripts that are still needed.
-// ** Copy the activation keys from the previous server.
-// Of course, it is always better to re-create the keys.
+Consider copying important or manually created data to a central place that can also be accessed by the new proxy.
 . Shut down the proxy.
-. Install a new proxy{nbsp}{productnumber}, following xref:install-proxy-unified.adoc[Proxy Installation].
+. Install a new proxy{nbsp}{productnumber}, following xref:install-proxy-uyuni.adoc[Uyuni Proxy Installation].
 . In the {productname} {webui} select the newly installed {susemgrproxy} and delete it from the systems list.
-[[step.at.proxy.migration3.replace.react]]
 . In the {webui}, create a reactivation key for the old proxy system: On the System Details tab of the old proxy click [guimenu]``Reactivation``.
```

Comment:
@mantel do we need this new reactivation key?

Reply:
The re-activation key is only needed if you do not want to lose the history of the machine. Else it will become a "new" one with a new ID.

Reply:
added this info also for uyuni